kano_qt_apps/lib/src/q_app_list.cpp

```diff
@@ -59,15 +59,25 @@ void QAppList::clean_q_app_list()
 }
 
 
-// TODO: Update the array as it changes to avoid calculating this multiple times
 QList<QVariant> QAppList::get_app_list()
 {
+    return get_app_list(true);
+}
+
+
+// TODO: Update the array as it changes to avoid calculating this multiple times
+QList<QVariant> QAppList::get_app_list(bool filter=true)
```

Comment:
Also, can we put the optional argument in the header where it is easier to find (also I'm surprised that it compiles this way because any file which includes the header doesn't know that the parameter is optional; presumably it is only because you have the other function too).